src/models/apigateway-base.ts

```diff
@@ -2,10 +2,20 @@ import DomainInfo = require("./domain-info");
 import ApiGatewayMap = require("./api-gateway-map");
 import DomainConfig = require("./domain-config");
 import {Client} from "@aws-sdk/smithy-client";
+import {ConfiguredRetryStrategy} from "@aws-sdk/util-retry";
 
 abstract class APIGatewayBase {
     public apiGateway: Client<any, any, any, any>;
 
+    public static getRetryStrategy() {
```

Comment:
Maybe this function shouldn't be part of the `APIGatewayBase` class since its pretty generic and could be used for any AWS API